### PR TITLE
feat(DiseasyActivity): add activity data

### PR DESCRIPTION
**Pull request by RasmusSkytte**
### Intent
This PR is part of the re-factoring of PR #30 which got too large due to scope-creep.
I am breaking that PR up into smaller chunks to make reviewing easier.

This PR does the following:
* The data set "dk_activity_units" was translated from Danish to English
* The data set "dk_reference_scenario" was translated from Danish to English
* The data set "contact_basis" was created as a replacement for old contact matrix data
  * It uses the `contactdata` package as source of contact matrices
  * It uses the U.S. Census Bureau as source of population data
* .rd tests set to exclude data sets

### Approach
1) The data set `contact_basis` was constructed using the same approach as the old contact matrix data but through the `contactdata` package we get projections for several geographical regions instead.
`contact_basis` is therefore a nested list where each element is analogous to the old data.

2) Our automatic tests of `.rd` files does not make sense for `.rd` files that document data. 
The tests ensures that .rd files have `\example` and `\value` fields, which are not required for data.
So these are now skipped.

3) Translation was done automatically but I used git versioning to ensure that only the text labels altered.
In fact, of all the 1600+ lines translated in `dk_activity_units`, there was a single occasion where the translator changed a numerical value.

4) Some variables were renamed
* `faWork` -> `social_distance_work`

### Known issues
* [ ] Test of `.rd` files fail as they are not yet fully documented. PR #59 takes care of this.

### Checklist

* [x] The PR passes all local unit tests
* [ ] I have documented any new features introduced
* [ ] If the PR adds a new feature, please add an entry in `NEWS.md`
* [x] A reviewer is assigned to this PR

### CODE REVIEW

File: data-raw/dk_activity_units.R [hunk 1]
```diff
@@ -0,0 +1,1624 @@
...
+    other = 0.07 + private_residual_other_add,
+    risk = 1
+  )
+
+
+
+################################## #
+dk_activity_units <- dk_activity_units |>
+  add_activity(
+    activity = "home_225",
```
Comment (LasseEngboChr) [inline data-raw/dk_activity_units.R:1560]: Rename home_225 -> home_22_5
Repeat for other activities for consistency

File: R/data.R [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+#' The contact matrices are provided for each of the four arenas: "Work", "Home", "School", "Other" in
+#' 5-year age groups.
+#'
+#' The `diseasy` SEIR models are configured to use a specific transformation of the contact matrices.
+#' To make the definition of "contact" matrix more clear, lets start with the definitions from this paper:
+#' https://www.medrxiv.org/content/10.1101/2020.02.16.20023754v2.full.pdf
+#'
+#' m_ij the raw contact matrix elements from age group i to age group j\cr
+#' c_ij the reciprocal contact matrix elements\cr
+#' w_i  the proportion of persons that fall into age group i\cr
```
Comment (LasseEngboChr) [inline R/data.R:16]:
```suggestion
#' w_i  the proportion of population that fall into age group i\cr
```

File: R/data.R [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+#'   *17*(7), e1009098. doi:10.1371/journal.pcbi.1009098
+#'   <https://doi.org/10.1371/journal.pcbi.1009098>.
+#' @keywords data
+NULL
+
+
+#' dk_activity_units
+#'
+#' This data set contains the "units" of activity that was developed for the Danish COVID-19 modelling efforts.
+#' Each "activity unit" specifies the amount of activity the for arenas: Home, Work, School, Other.
```
Comment (LasseEngboChr) [inline R/data.R:65]:
```suggestion
#' Each "activity unit" specifies the amount of activity for the four arenas: Home, Work, School, Other.
```

File: R/data.R [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+#' @keywords data
+NULL
+
+
+#' dk_reference_scenario
+#'
+#' This data set contains the combination of "activity units" (see dk_activity_units for details) that form a
+#' description of the societal activity in Denmark through out the COVID-19 pandemic.
+#'
+#' The data set consists of a `tibble` with four columns:
```
Comment (LasseEngboChr) [inline R/data.R:89]: Why a 'tibble' instead of a 'data.frame'?

**Review by LasseEngboChr — CHANGES_REQUESTED** [review]

File: R/data.R (hunk 4 above)
Comment (RasmusSkytte) [reply R/data.R:89]: Mostly becase "tibble" has better printing / quality of life features.

In Rstudio, you can compare 
```
> mtcars

                     mpg cyl  disp  hp drat    wt  qsec vs am gear carb
Mazda RX4           21.0   6 160.0 110 3.90 2.620 16.46  0  1    4    4
Mazda RX4 Wag       21.0   6 160.0 110 3.90 2.875 17.02  0  1    4    4
Datsun 710          22.8   4 108.0  93 3.85 2.320 18.61  1  1    4    1
Hornet 4 Drive      21.4   6 258.0 110 3.08 3.215 19.44  1  0    3    1
Hornet Sportabout   18.7   8 360.0 175 3.15 3.440 17.02  0  0    3    2
Valiant             18.1   6 225.0 105 2.76 3.460 20.22  1  0    3    1
Duster 360          14.3   8 360.0 245 3.21 3.570 15.84  0  0    3    4
Merc 240D           24.4   4 146.7  62 3.69 3.190 20.00  1  0    4    2
Merc 230            22.8   4 140.8  95 3.92 3.150 22.90  1  0    4    2
Merc 280            19.2   6 167.6 123 3.92 3.440 18.30  1  0    4    4
Merc 280C           17.8   6 167.6 123 3.92 3.440 18.90  1  0    4    4
Merc 450SE          16.4   8 275.8 180 3.07 4.070 17.40  0  0    3    3
Merc 450SL          17.3   8 275.8 180 3.07 3.730 17.60  0  0    3    3
Merc 450SLC         15.2   8 275.8 180 3.07 3.780 18.00  0  0    3    3
Cadillac Fleetwood  10.4   8 472.0 205 2.93 5.250 17.98  0  0    3    4
Lincoln Continental 10.4   8 460.0 215 3.00 5.424 17.82  0  0    3    4
Chrysler Imperial   14.7   8 440.0 230 3.23 5.345 17.42  0  0    3    4
Fiat 128            32.4   4  78.7  66 4.08 2.200 19.47  1  1    4    1
Honda Civic         30.4   4  75.7  52 4.93 1.615 18.52  1  1    4    2
Toyota Corolla      33.9   4  71.1  65 4.22 1.835 19.90  1  1    4    1
Toyota Corona       21.5   4 120.1  97 3.70 2.465 20.01  1  0    3    1
Dodge Challenger    15.5   8 318.0 150 2.76 3.520 16.87  0  0    3    2
AMC Javelin         15.2   8 304.0 150 3.15 3.435 17.30  0  0    3    2
Camaro Z28          13.3   8 350.0 245 3.73 3.840 15.41  0  0    3    4
Pontiac Firebird    19.2   8 400.0 175 3.08 3.845 17.05  0  0    3    2
Fiat X1-9           27.3   4  79.0  66 4.08 1.935 18.90  1  1    4    1
Porsche 914-2       26.0   4 120.3  91 4.43 2.140 16.70  0  1    5    2
Lotus Europa        30.4   4  95.1 113 3.77 1.513 16.90  1  1    5    2
Ford Pantera L      15.8   8 351.0 264 4.22 3.170 14.50  0  1    5    4
Ferrari Dino        19.7   6 145.0 175 3.62 2.770 15.50  0  1    5    6
Maserati Bora       15.0   8 301.0 335 3.54 3.570 14.60  0  1    5    8
Volvo 142E          21.4   4 121.0 109 4.11 2.780 18.60  1  1    4    2
```

verus

```
> tibble::tibble(mtcars)

# A tibble: 32 x 11
     mpg   cyl  disp    hp  drat    wt  qsec    vs    am  gear  carb
   <dbl> <dbl> <dbl> <dbl> <dbl> <dbl> <dbl> <dbl> <dbl> <dbl> <dbl>
 1  21       6  160    110  3.9   2.62  16.5     0     1     4     4
 2  21       6  160    110  3.9   2.88  17.0     0     1     4     4
 3  22.8     4  108     93  3.85  2.32  18.6     1     1     4     1
 4  21.4     6  258    110  3.08  3.22  19.4     1     0     3     1
 5  18.7     8  360    175  3.15  3.44  17.0     0     0     3     2
 6  18.1     6  225    105  2.76  3.46  20.2     1     0     3     1
 7  14.3     8  360    245  3.21  3.57  15.8     0     0     3     4
 8  24.4     4  147.    62  3.69  3.19  20       1     0     4     2
 9  22.8     4  141.    95  3.92  3.15  22.9     1     0     4     2
10  19.2     6  168.   123  3.92  3.44  18.3     1     0     4     4
# i 22 more rows
# i Use `print(n = ...)` to see more rows
```

File: data-raw/dk_activity_units.R (hunk 1 above)
Comment (RasmusSkytte) [reply data-raw/dk_activity_units.R:1560]: Renaming home_225 -> home_22_5pp

(In the rest of the activity unites, we often use "pp" to indicate percentage changes)

File: data-raw/contact_basis.R [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+  # https://www.census.gov/data/developers/about/terms-of-service.html
+
+  # Alternative data source is here:
+  # https://population.un.org/wpp/Download/Standard/Population/
+  idbzip <- "https://www.census.gov/data-tools/demo/data/idb/dataset/idbzip.zip"
+  curl::curl_fetch_disk(idbzip, file.path(tempdir(), "idbzip.zip"))
+  idb1yr <- readr::read_delim(unz(file.path(tempdir(), "idbzip.zip"), "idbsingleyear.txt"),
+                              delim = "|", show_col_types = FALSE)
+
+  # Get 1-year age-group data for Denmark
```
[the remark this answers is not in the document]
Comment (RasmusSkytte) [reply data-raw/contact_basis.R]: Done

File: data-raw/contact_basis.R [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+  pop_ref_1yr <- idb1yr |>
+    dplyr::rename_with(tolower) |>
+    dplyr::filter(`#yr` == 2020, .data$sex == 0) |>
```
[the remark this answers is not in the document]
Comment (RasmusSkytte) [reply data-raw/contact_basis.R]: More comments added to this section

**Review by LasseEngboChr — APPROVED** [review]

**Pull request merged**